### ISSUE TITLE
VerifyAllCodeStyleFixersAreSupportedByCodeCleanup tests are failing

### ISSUE THREAD

**Issue by stephentoub** [dotnet/roslyn#65566]
**Version Used**: 
abfab568aadd0406c6b78722790c1276244f3b8a

**Steps to Reproduce**:
Run the VerifyAllCodeStyleFixersAreSupportedByCodeCleanup test with main.  PRs are failing with it.

**Expected Behavior**:
Passes

**Actual Behavior**:
```
Assert.Equal() Failure
Expected: 38
Actual:   39

Assert.Equal() Failure
Expected: 75
Actual:   76
```

**Comment by Youssef1313** [comment]
https://github.com/dotnet/roslyn/pull/65564

**Comment by arkalyanms** [comment]
Tagging @mavasani to buddy land this PR.

**Comment by jjonescz** [comment]
Fixed in #65575.
